utils/parametric/protos/apm_test_client.proto

```diff
@@ -78,6 +90,35 @@ message FinishSpanArgs {
 }
 message FinishSpanReturn {}
 
+message SpanNameArgs {
+  uint64 span_id = 1;
+}
+message SpanNameReturn {
+  string name = 1;
+}
+
+message SpanResourceArgs {
+  uint64 span_id = 1;
+}
+message SpanResourceReturn {
+  string resource = 1;
+}
+
+message SpanMetaArgs {
+  uint64 span_id = 1;
+  string key = 2;
+}
+message SpanMetaReturn {
+  string value = 1;
```

Comment:
Question: Is meta only a string or can it be anything else?

Reply:
In PHP, although at runtime essentially any type can be assigned as a value, only strings, numbers (int or floats), boolean, null, arrays, and enum are serialized (I believe this list is exhaustive). Objects aren't serialized. Eventually, everything gets serialized into a string when flushed.

I believe `AttrVal` should be OK there (+ null/None?)